src/librustc/metadata/decoder.rs

```diff
@@ -126,6 +126,7 @@ enum Family {
     TupleVariant,          // v
     StructVariant,         // V
     Impl,                  // i
+    DefTrait,              // d
```

Comment:
I'm terrible with naming things. If we rename this to `DefaultImpl`, we should probably also rename the `Item_` as `DefaultImpl` instead of `DefaultTrait`